servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -69,6 +70,10 @@
   private volatile boolean mRunning;
   /** The id of the worker */
   private long mWorkerId;
+  /** The thread pool to remove block */
+  private static final int DEFAULT_BLOCK_REMOVER_POOL_SIZE = 10;
```

Comment:
Better move this constant declaration up according to the [recommended coding style](http://tachyon-project.org/v0.7.0/Contributing-to-Tachyon.html)